api/schemas/input/group.json

```diff
@@ -4,15 +4,15 @@
   "type": "object",
   "properties": {
     "_id":              {
-                          "maxLength": 32,
+                          "maxLength": 64,
                           "minLength": 2,
                           "pattern": "^[0-9a-z][0-9a-z.@_-]{0,30}[0-9a-z]$",
                           "title": "ID",
                           "type": "string"
                         },
     "name":             {
-                          "maxLength": 32,
-                          "minLength": 2,
+                          "maxLength": 64,
+                          "minLength": 1,
```

Comment:
What's the motivation behind this change (and the similar one further up the file)?